pombola/south_africa/templatetags/za_people_display.py

```diff
@@ -0,0 +1,29 @@
+from django import template
+
+register = template.Library()
+
+NO_PLACE_ORGS = ('parliament', 'national-assembly', )
+MEMBER_ORGS = ('parliament', 'national-assembly', )
+
+
+@register.assignment_tag()
+def should_display_place(organisation):
+    should_display = True
+
+    if organisation.slug in NO_PLACE_ORGS:
+        should_display = False
+
+    return should_display
+
+
+@register.assignment_tag()
+def should_display_position(organisation, position):
+    should_display = True
+
+    if organisation.slug in MEMBER_ORGS and str(position) in ('Member',):
+        should_display = False
+
+    if 'ncop' in organisation.slug and str(position) in ('Delegate',):
```

Comment:
What is the `'ncop' in organisation.slug` trying to catch as well as the exact slug `ncop`? (If there is other stuff, I'd add a comment about what it's trying to match, and possibly make it a regexp match so it's more precise - if there aren't other slugs, I'd prefer an `==` match.

I'd make the second part of both conditionals here: `unicode(position) in (u'Delegate',)` with a similar change above in case there are non-ASCII characters in the position title's name. (I don't think it'll fail in any bad way in practice at the moment, but it might help someone in future changing this code.)